payments-model/src/main/java/com/stripe/android/model/ConsumerPaymentDetails.kt

```diff
@@ -55,6 +55,10 @@ data class ConsumerPaymentDetails(
                 expiryYear = expiryYear
             )
 
+        val availableNetworks = networks
+            .map { CardBrand.fromCode(it) }
+            .filter { it != CardBrand.Unknown }
```

Comment:
Can we make this a computed property?

Reply:
https://github.com/stripe/stripe-android/pull/10715/commits/8fe4434e5b46e4dd6c02529307d904b8646c19d1